### PR TITLE
[UNOMI-722]: Update Yauaa to 7.13.0

### PR DESCRIPTION
**Pull request by nielsbasjes**
Merge request https://github.com/apache/unomi/pull/557 introduced the need for an unexpected extra `<Import-Package>` setting because of a dependency of Yauaa.
In the just released version of Yauaa this should be fixed.

----

**Please** following this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/UNOMI) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Format the pull request title like `[UNOMI-XXX] - Title of the pull request`
 - [x] Provide integration tests for your changes, especially if you are changing the behavior of existing code or adding
       significant new parts of code.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why. 
       Copy the description to the related JIRA issue
 - [x] Run `mvn clean install -P integration-tests` to make sure basic checks pass. A more thorough check will be 
        performed on your pull request automatically.
 
Trivial changes like typos do not require a JIRA issue (javadoc, project build changes, small doc changes, comments...). 
 
If this is your first contribution, you have to read the [Contribution Guidelines](https://unomi.apache.org/contribute.html)

If your pull request is about ~20 lines of code you don't need to sign an [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) 
if you are unsure please ask on the developers list.

To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Comment by jkevan** [comment]
Thx for the contribution again.
Is the PR available for review ? can you move it to review so we could merge it ? It's currently in draft.
(I rerun the tests that failed, I think it was a flaky run)

**Comment by nielsbasjes** [comment]
> Thx for the contribution again. Is the PR available for review ? 

I released a new version yesterday. Received a few more edge cases today that I want to fix. 
As soon as I have that in I'll update it.

> can you move it to review so we could merge it ? It's currently in draft. 

I know, I put it in draft because I was unsure if it would pass the build ... and then it didn't.
Yes, I will put it in ready for review do when ready.

> (I rerun the tests that failed, I think it was a flaky run)

Aha, that explains it. I completely did not understand what caused the CI run to fail. I went looking what I did wrong and couldn't figure it out.
Thanks for clarifying.

**Comment by jkevan** [comment]
Alright, yes the tests passed on the second run, so everything looks fine.

Thank you !

**Comment by nielsbasjes** [comment]
@jkevan I've updated to the latest version and marked it ready for review.

**Review by jkevan — APPROVED** [review]

File: plugins/request/pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>nl.basjes.parse.useragent</groupId>
             <artifactId>yauaa</artifactId>
-            <version>7.11.0</version>
+            <version>7.12.0</version>
```
Comment (jkevan) [inline plugins/request/pom.xml:49]: Shouldn't be 7.13.0 version here ?

Comment (nielsbasjes) [reply plugins/request/pom.xml:49]: Yes

Comment (nielsbasjes) [reply plugins/request/pom.xml]: Sorry about that. Fixed.

**Review by jkevan — APPROVED** [review]

**Comment by jkevan** [comment]
Thanks for contributing the update of the lib.
Best regards.

I will merge it now.

**Pull request merged**